docs/image-acquisition/image-file.md

```diff
@@ -0,0 +1,29 @@
+Creating a frame out of an image file is pretty straightforward:
+```py
+from landingai.pipeline.frameset import Frame
+
+frame = Frame.from_image("/path/to/your/image.jpg")
+frame.resize(width=512, height=512)
+frame.save_image("/tmp/resized-image.png")
```

Comment:
here is missing the tooltips numbers right ?

Reply:
Good catch!